Homework/Hung_Tran/217_contains_duplicate.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @param {number[]} nums
+ * @return {boolean}
+ */
+var containsDuplicate = function (nums) {
```

Comment:
please submit on LC first, I think this approach will fail to pass all test cases

Reply:
I've submitted and success:
![image](https://user-images.githubusercontent.com/17242600/156930970-72dec5e5-98ba-47ee-ab91-683a7f070bc0.png)

Reply:
Ah I see. The runtime is quite high. Can you try to improve? I expect its runtime to be O(N)

Reply:
Ok got it